src/it/java/com/google/checkstyle/test/chapter5naming/rule526parameternames/ParameterNameTest.java

```diff
@@ -43,36 +49,63 @@ protected String getPath(String fileName) throws IOException {
 
     @BeforeClass
     public static void setConfigurationBuilder() throws CheckstyleException {
-        checkConfig = getCheckConfig("ParameterName");
-        format = checkConfig.getAttribute("format");
+        final List<Configuration> configs = getCheckConfigs("ParameterName");
+
+        Assert.assertEquals(configs.size(), 2);
+
+        privConfig = configs.get(0);
```

Comment:
@rnveach , yes order of check to execute is not strick and order of violations is not strict too.